### PR TITLE
Modified tutorial 01 console introduction string

### PR DESCRIPTION
**Pull request by Kurara**
# Trac ticket number

There is no ticket assigned to this pr. This modification came from django con workshop in a [separated excel file](https://docs.google.com/spreadsheets/d/16UTGwtAoOwznc46cszbwAHU9xbukXnnpwG-faE94Rw8/edit#gid=0)

# Branch description
There was a line reffering to unix console. It has been modified by adding a comment extended that other consoles may exists.

# Checklist
- [x] This PR targets the `main` branch. 
- [x] The commit message is written in past tense, mentions the ticket number, and ends with a period.
- [ ] I have checked the "Has patch" ticket flag in the Trac system.
- [ ] I have added or updated relevant tests.
- [ ] I have added or updated relevant docs, including release notes if applicable.
- [ ] I have attached screenshots in both light and dark modes for any UI changes.

### CODE REVIEW

File: docs/intro/tutorial01.txt [hunk 1]
```diff
@@ -14,7 +14,8 @@ It'll consist of two parts:
 
 We'll assume you have :doc:`Django installed </intro/install>` already. You can
 tell Django is installed and which version by running the following command
-in a shell prompt (indicated by the $ prefix):
```
Comment (felixxm) [inline docs/intro/tutorial01.txt:17]: `$` in this refers to the `$` in the below example. I'd change this.

Comment (felixxm) [reply docs/intro/tutorial01.txt:17]: It's only an explanation of what is `$` in the above example.

Comment (Kurara) [reply docs/intro/tutorial01.txt:17]: The problem is that there is no $ in windows example.

Comment (carltongibson) [reply docs/intro/tutorial01.txt:17]: OK, let's reopen to allow a longer review.

**Comment by Kurara** [comment]
Since there is no activity and it's not a big change I'm closing the pr.

**Comment by nessita** [comment]
Hey @Kurara! The lack of activity was due to the Fellows handling other tasks with higher priority (fixing of release blocker issues, security reports, Django 5.1 releases).

With the Django 5.1 final release just issued, I was planning on restarting work on these DjangoCon Europe PRs. If you want, you can re-open and I'll review. But it's also OK if you wish to keep it closed.

Thanks!

**Pull request closed without merging**